README.md

```diff
@@ -1,46 +1,123 @@
 
-using post_path(@post.id)
+* **Column 4** - This column shows the controller and action with the syntax of ```controller#action```.
 
-using post_path(@post)
+One of the other nice things about utilize route helper methods is that they create predictable names for the methods. Once you get into day to day Rails development you will only need to run ```rake routes``` to find custom paths. Let's imagine that you takeover a legacy Rails application that was built with traditional routing conventions. If you see CRUD controllers for: newsletters, students, sales, offers, and coupons; you don't have to lookup the routes to know that you could call the index URLs for each resource below:
+
+* Newsletters - ```newsletters_path```
+
+* Students - ```students_path```
+
+* Sales - ```sales_path```
+
+* Offers - ```offers_path```
+
+* Coupons - ```coupons_path```
+
+
+## link_to Method
+
+All of our tests are currently passing, let's create a new Capybara spec. The scenario we will be building is to ensure that a link from the index page will point to that post's respective show page view template. The scenario is below:
+
+```ruby
+scenario 'index page links to post page' do
+  second = FactoryGirl.create(:second_post)
+  visit posts_path
+  expect(page).to have_link(second.title, href: post_path(second))
+end
+```
+
+This matcher will fail since our index page doesn't currently link to the show page. To fix this let's update the index page like so:
+
+```ERB
+<% @posts.each do |post| %>
+  <div><%= link_to post.title, "/posts/#{post.id}" %></div>
+<% end %>
+```
+
+This works and gets the tests passing, however it can be refactored. Instead of hardcoding the path and using string interpolation, let's using ```post_path``` and pass in the post argument.
+
+```ERB
+<% @posts.each do |post| %>
+  <div><%= link_to post.title, post_path(post.id) %></div>
+<% end %>
+```
+
+This is much better, but to be thorough, let's make one last refactor: Rails is smart enough to know that if you pass in the ```post``` object as an argument, it will automatically use the ID attribute, so we'll use this implementation code:
+
+```ERB
+<% @posts.each do |post| %>
+  <div><%= link_to post.title, post_path(post) %></div>
+<% end %>
+```
+
+If you run the tests now you'll see that they're all still passing.
+
+We're using the ```link_to``` method to automatically create an HTML ```a``` tag. Now all of the tests are passing. If you open the browser and inspect the HTML element of the link you would see the following:
+
+![Link To](http://reif.io/lib/flatiron/link_to.png)
+
+As you will see, even those we never added HTML code for the link, such as: ```<a href="..."></a>``` the ```link_to``` method rendered the correct tag for us.
+
+
+## Using the :as option
+
+If for any reason you don't like the naming structure for the methods or paths you can customize them quite easily. Let's say we want to change the show path from:
+
+```
+/posts/42
```

Comment:
Instead of a change that breaks REST in like a subtle way, can you do something like instead of having /users/new we want the route to be /register. That url is better then /users/new, where the /post/42 isn't necessarily better